### DISCUSSION TITLE
Lindblad evolution

### DISCUSSION THREAD

**Discussion by ProThicc** [jcmgray/quimb#221] [General]
Hello,
Is the Lindblad evolution not implemented or is there a way around to use it somewhere in the code?

https://github.com/jcmgray/quimb/blob/main/quimb/evo.py#L557
From what I could understand, this line sets the system to closed by default, but I am not sure if somewhere down the code there is a way to do the open system evolution

Thanks,

Pratik

**Comment by jcmgray** [comment]
Yes it looks like there is no way to supply the Lindblad operators currently to the `Evolution` class, thus it is always closed, likely I planned to add open evolution at some point but never needed it.

**Comment by ProThicc** [comment]
Gotcha, I was trying to write a QUIMB based code to use Lindblad operators and do a Monte-carlo wavefunction simulation, similar to QuTiP but using parallel execution of trajectories for each node using mpi. If it works, I can help with the development of this feature.
